WcaOnRails/app/controllers/api/v0/users_controller.rb

```diff
@@ -29,37 +23,43 @@ def show_user_by_wca_id
   end
 
   def permissions
-    if current_user
-      if stale?(current_user)
-        render json: current_user.permissions
-      end
-    else
-      render status: :unauthorized, json: { error: I18n.t('api.login_message') }
+    require_user!
+    if stale?(current_user)
+      render json: current_user.permissions
     end
   end
 
+  def personal_records
+    require_user!
+    return render json: { single: [], average: [] } unless current_user.wca_id.present?
+    person = Person.includes(:user, :ranksSingle, :ranksAverage).find_by_wca_id!(current_user.wca_id)
```

Comment:
Where does the `:user` in the includes come from? As far as I can tell, `ranksSingle` and `ranksAverage` should be enough to compute the return value below.

Reply:
it comes from the way it's done in other places of the code where the ranksSingle and ranksAverage are queried, but I guess it's not needed here